Modules/ChequeModule/Views/ChequeView.xaml

```diff
@@ -17,7 +19,17 @@
         </ResourceDictionary>
     </UserControl.Resources>
 
-    <Grid Background="{StaticResource Secondary}">
+    <Grid Margin="10">
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="*"/>
+            <ColumnDefinition Width="*"/>
+        </Grid.ColumnDefinitions>
+        <ContentControl Grid.Column="0" prism:RegionManager.RegionName="ChequeFormContentRegion"/>
+        <ContentControl Grid.Column="1" prism:RegionManager.RegionName="ChequePreviewContentRegion"/>
+    </Grid>
+    
+    <!--TODO: Have to move the related code design to respective region view-->
```

Comment:
Pending for moving the respective codes:

Form control -> ChequeFormView
Canva control -> ChequePreviewView

I have to discuss with @Yawhou for these changes.

Reply:
Resolved